apps/meteor/app/file-upload/server/methods/sendFileMessage.ts

```diff
@@ -177,6 +185,8 @@ export const sendFileMessage = async (
 			groupable: Match.Optional(Boolean),
 			msg: Match.Optional(String),
 			tmid: Match.Optional(String),
+			t: Match.Optional(String),
+			e2e: Match.Optional(String),
```

Comment:
Can we create an API test case for this? Just to ensure E2EE properties are being saved correctly.

Reply:
Done!